packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -707,6 +707,44 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for SystemData
+    ///
+
+    // RPC Code: RPC-SystemData-V1-01 and RPC-SystemData-V1-02
+    // Note that this rule is normally already covered when the common types Resource is used.
+    // Here we are still checking for usage of common types system data for the scenarios where
+    // RP teams have already defined Resource and are unable to switch to the common types resource
+    // because it would cause breaking changes.
+    SystemDataDefinitionsCommonTypes: {
+      description: "Must use the schema provided in the common types for SystemData.",
+      message: "{{description}}",
+      severity: "error",
+      resolved: false,
+      formats: [oas2],
+      given: "$.definitions.*.properties.[systemData,SystemData].$ref",
+      then: {
+        function: pattern,
+        functionOptions: {
+          match: ".*/common-types/resource-management/v\\d+/types.json#/definitions/systemData",
+        },
+      },
+    },
+
+    // RPC Code: RPC-SystemData-V1-01 and RPC-SystemData-V1-02
+    // Ensure systemData is not in the properties bag
+    SystemDataInPropertiesBag: {
+      description: "System Data must be defined as a top-level property, not in the properties bag.",
```

Comment:
```suggestion
      description: "SystemData must be defined as a top-level property, not in the properties bag.",
```

Reply:
Changed it to "System data" to match the other rule's description.